contracts/token/IMinter.sol

```diff
@@ -1,14 +1,23 @@
 pragma solidity ^0.4.17;
 
+import "../IController.sol";
 
+
+/**
+ * @title Minter interface
+ */
 contract IMinter {
-    event NewInflation(uint256 inflation);
-    event SetCurrentRewardTokens(uint256 mintableTokens);
+    // Events
+    event SetCurrentRewardTokens(uint256 currentMintableTokens, uint256 currentInflation);
```

Comment:
Collapsed `NewInflation` and `SetCurrentRewardTokens` into the same event that is fired with both the current mintable tokens and current inflation for the round so a client would only need to listen to a single event if it cares about these values as opposed to listening for two events